### PR TITLE
Improve typescript config docs

### PR DESCRIPTION
**Pull request by web-devel**
## What I did

Updated typescript configuration docs so that stories made with typescript picked up by default.

### CODE REVIEW

File: docs/src/pages/configurations/typescript-config/index.md [hunk 1]
```diff
@@ -69,6 +69,15 @@ The above example shows a working config with the TSDocgen plugin also integrate
 
 This is for the default configuration where `/stories` is a peer of `src`. If you have them all in just `src` you may wish to replace `"rootDirs": ["src", "stories"]` above with `"rootDir": "src",`.
 
+## Import tsx stories
+
+Change `config.js` inside the Storybook config directory (by default, it’s `.storybook`) to import stories made with Typescript:
```
Comment (igor-dv) [inline docs/src/pages/configurations/typescript-config/index.md]: BTW, it can be also `config.ts`

**Review by danielduan — APPROVED** [review]

**Review by ndelangen — APPROVED** [review]

**Review by ndelangen — APPROVED** [review]

**Review by web-devel — COMMENTED** [review]
I don't think renaming `config.js` to `config.ts` is a good idea since `.storybook` directory is not included in ts compiler (so it won't work out of the box without more changes in this guide). 

Also it seems `@storybook/react` looks exactly for `config.js` :
> Error: => Create a storybook config file in "./.storybook/config.js".

I suggest leaving `config.js` as is.

**Comment by igor-dv** [comment]
If you configured Storybook to work with TS, it will also be able to load `config.ts` and `addons.ts`.

**Comment by web-devel** [comment]
> If you configured Storybook to work with TS, it will also be able to load config.ts and addons.ts.

If you follow this particular guide it won't. At lest we also have to change `tsconfig.json` to include files in `.storybook`.

**Comment by igor-dv** [comment]
Can you elaborate how did you change it? Also, what is your version of Storybook?

**Comment by web-devel** [comment]
@igor-dv I used `latest` which is `v3.4.10`, which has `config.js` lookup hardcoded: https://github.com/storybooks/storybook/blob/release/3.4/app/react/src/server/config.js#L26
It seems things changed in master though

**Comment by ndelangen** [comment]
What's the next course of action then? Do we add a remark about typescript support to the docs?

**Comment by web-devel** [comment]
@igor-dv @ndelangen I was able to test it with latest alpha and it works (config.ts/addons.ts).

**Comment by ndelangen** [comment]
I'll turn this into a non-patch, since this now documents 4.0.0 behavior

**Pull request merged**